content/app/github/wrappers/file-element-wrapper.js

```diff
@@ -37,6 +39,18 @@ define("content/app/github/wrappers/file-element-wrapper", function(require, exp
 		 */
 		constructor : function (domElement) {
 			FileElementWrapper.$super.call(this, domElement);
+			this._fileName = null;
+			this._init();
+		},
+
+		/**
+		 * File name.
+		 * @type {String}
+		 */
+		fileName : {
+			get : function() {
+				return this._fileName;
+			}
 		},
```

Comment:
I didn't think of getter itself, I was thinking about place where are you getting file name. My initial idea was to have separate FileHeaderWrapper and FileDataWrapper, so title will be read from header wrapper and lines from data wrapper. But at this point it will make complex structure without good reason. So maybe I should move functionality from file data wrapper in this class and remove that wrapper. Let's discuss offline.